_posts/shaders/2019-08-27-guide_the-pbr-shader.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: The PBR Shader
-permalink: /guide/pbr-shader/
-category: guide
+permalink: /shader/pbr-shader/
+category: shader
 tags:
   - pbr
```

Comment:
Would be good to add `shader` tag since the modulate has it. Not on the parallax corrected cubemaps guide though as that is not a shader technically.

Reply:
Resolved in [`36540aa` (#233)](https://github.com/momentum-mod/docs/pull/233/commits/36540aad6a67d20708e15095329ae57c3da3fe72)